translation/en/RedirectedResources/assets/abdata/communication/info_20/optiondisplayitems_36/translation.txt

```diff
@@ -1,74 +1,36 @@
-はい=Yes
-いいえ=No
+はい=Yes.
+いいえ=No.
 いいよ！　問題なし！=It's OK! No problem!
-それは流石にちょっと……=That much is a bit...
-好きだよ。ブタも結構可愛いよね=I like them. Pigs is also pretty cute, right?
-苦手だね。ブタも含めて全体的に=Not really. Including pigs、on the whole.
+それは流石にちょっと……=That's really a bit...
+好きだよ。ブタも結構可愛いよね=I like them. Pigs too, they are pretty cute in their own way, no?
+苦手だね。ブタも含めて全体的に=I'm bad with them. Especially pigs in general.
 がっつり食べるよ！=I eat a lot!
-実は小食だったりする=I don't eat much
-作れるよ=I can
-うーん、作れない……=Yeah、I cant...
-大好きなので、無駄に動いてます！=I love it anyways!
-いや、できる事なら動きたくない=Nah, I don't want to move if I don't have to
-辛辣だね……してるよ=Harsh words... But I do my studies.
-その通り、全然してない=Exactly、not at all.
-心外だな、結構頑張ってるよ=That hurt, I'm actually trying pretty hard
-お察しの通り、壊滅的です=As you guessed、it's catastrophic
-うん、どこまでも付いてくよ！=Yeah、I'll follow you anywhere!
-実は、ちょっと辛くなってきてる……=To tell you the truth、it's getting a little harder...
-ブラック派=Black
-加糖派=With sugar
-平気だよ=Yeah
-苦手かなー=Probably not...
-俺は甘いもの大好き、スイーツ男子！=I love sweet things!
-あんまり好きじゃないかな=Not really
-大きいのが=Big ones
-程よいのが=Moderate ones
-小さいのが=Small ones
-高いのが=Tall ones
-ぽっちゃりしているのが=Chubby ones
-痩せているのが=Thin ones
-うーん、そうかも？=Yeah、probably?
-そんな事ないよ=That's not true at all
-バッチリ！　１０年先まで！=Exactly! I plan 10 years ahead!
-実際何も考えてないです……=Actually I don't think much about it...
-それはまぁ……してみたいかな=Well... I think I'd like to try
-いや、それはないかな=Nah、rather not
-
```

Comment:
This part of new dump got removed in the process, it should be added back

Reply:
fixed